docs/app/pages/components/components-sections/tabs/card-tabs/snippets/app.html

```diff
@@ -0,0 +1,35 @@
+<ux-card-tabset [position]="position">
+
+    <ux-card-tab class="card" *ngFor="let tab of tabs">
+        <h3 class="card-title">{{ tab.title }}</h3>
+        
+        <div class="card-image" [ngClass]="tab.image"></div>
+        
+        <h3 class="card-value">{{ tab.value }}<small>{{ tab.unit }}</small></h3>
+        <h4 class="card-subtitle">{{ tab.subtitle }}</h4>
+
+        <ng-template uxCardTabContent>
+            <h4>{{ tab.title }}</h4>
+            <p>{{ tab.content }}</p>
+        </ng-template>
+    </ux-card-tab>
+
+</ux-card-tabset>
+
+<hr>
+
+<div class="row uxd-customize-example">
```

Comment:
I was thinking about this and I don't think we should include the whole "Customize" section in the code snippets. It adds a lot of code that isn't relevant to the component itself, and could potentially be confusing. The idea is to allow experimentation with the inputs, and when this is launched in plunker that can be done by modifying the variable values directly.